templates/wait-for-port.conf.erb

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStartPost=-/bin/bash -c 'while ! nc -z 127.0.0.1 <%= scope['qpid::ssl_port'] %>; do sleep 1; done'
```

Comment:
I do wonder about the `-` before `/bin/bash`. Why is the exit code irrelevant here? Shouldn't it eventually return true?

Reply:
And you think I still know why I added it yesterday? Yes, it will probably return 0 - or end up in an endless loop.

Reply:
Ah, I *think* I wanted to add a timeout to this, and that would produce a non-zero code. But I wanted to ignore it, just for the case where maybe we try to poke the wrong interface or something.

Reply:
So it'd be smth like:
```
ExecStartPost=-/usr/bin/timeout 60s /bin/bash >…
```

(timeout is in coreutils, no need to add a dep here :))

Reply:
Doesn't the systemd unit itself have a timeout already?

Reply:
droped the -